sunpy/net/dataretriever/tests/test_vsm.py

```diff
@@ -0,0 +1,49 @@
+"""
+This module tests SOLIS VSM Client.
+"""
+#This module was developed with funding provided by
+#the Google Summer of Code 2016.
+import pytest
+
+from astropy import units as u
+from sunpy.net.vso.attrs import Time, Instrument, Physobs, Wavelength
+from sunpy.net.dataretriever.downloader_factory import UnifiedResponse
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+import sunpy.net.dataretriever.sources.vsm as vsm
+
+VClient = vsm.VSMClient()
+
+trange = Time('2014/6/1', '2014/6/4')
```

Comment:
makei it capital, so we know is a global variable for all the functions: `TRANGE = ...`